### PR TITLE
Wrist Layer Toggling

### PR DESCRIPTION
**Pull request by Geevies**
* You can now set wristwear to layer under your uniform, over your uniform, or over your suit.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by DreamySkrell** [comment]
does this not allow wearing wristwear on top of voidsuits for example
kind of silly
probably not worth the effort to not allow that

File: code/modules/client/preference_setup/loadout/items/wrists.dm [hunk 1]
```diff
@@ -31,3 +35,37 @@
...
+
+/*
+	Wristwear Layer Adjustment
+*/
+
+#define WRISTS_UNDER "Under Uniform"
+#define WRISTS_OVER_UNIFORM "Over Uniform"
+#define WRISTS_OVER_SUIT "Over Suit"
+
+var/datum/gear_tweak/wrist_layer/gear_tweak_wrist_layer = new()
```
Comment (FluffyGhoster) [inline code/modules/client/preference_setup/loadout/items/wrists.dm:47]: Make this a GLOB var if you need a global variable

**Review by FluffyGhoster — CHANGES_REQUESTED** [review]

**Comment by Geevies** [comment]
> does this not allow wearing wristwear on top of voidsuits for example kind of silly

addressed this here: https://github.com/Aurorastation/Aurora.3/pull/18949

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**